Angeline Polidano/week_1/calculator/index.html

```diff
@@ -0,0 +1,14 @@
+<html lang="\" dir="ltr">
+  <head>
+    <meta charset ="utf-8">
+    <script src="js/main.exercises.js"></script>
```

Comment:
This all looks correct, only nitpick would be to avoid using dots in file names in the future, other than the one at the end. I would just call this 'exercises.js. instead of 'main.exercises.js'